kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/BatchRequest.java

```diff
@@ -110,6 +111,14 @@ public void setArgs(List<String> args) {
     this.args = args;
   }
 
+  public Map<String, String> getExtraResourcesMap() {
+    return extraResourcesMap;
```

Comment:
return emptyMap if it is null

Reply:
extraResourcesMap was initialed as empty map in L34. (https://github.com/apache/kyuubi/pull/6335/files/d35219f9f388a1fb91e12c2f008730e5aabf015b#diff-5dffcdb403904677ed5c8e8d4c997513aa236dc4cfa828a1e0dad1d9bec93644L34)